src/thunderbots/software/ai/primitive/stop_primitive.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "ai/primitive/primitive.h"
+
+class StopPrimitive : public Primitive
+{
+    public:
+     static const std::string PRIMITIVE_NAME;
+
+    /**
+     * Creates a new Stop Primitive
+     *
```

Comment:
It's helpful to also include a quick description of what the primitive does, eg "stops the robot in place, with an option to coast to a stop rather than stop immediately"